bridge/opencensus/internal/ocmetric/metric_test.go

```diff
@@ -935,7 +935,12 @@ func TestConvertMetrics(t *testing.T) {
 		t.Run(tc.desc, func(t *testing.T) {
 			output, err := ConvertMetrics(tc.input)
 			if !errors.Is(err, tc.expectedErr) {
-				t.Errorf("ConvertMetrics(%+v) = err(%v), want err(%v)", tc.input, err, tc.expectedErr)
+				t.Errorf(
+					"ConvertMetrics(%+v) = err(%v), want err(%v)",
+					tc.input,
+					err,
+					tc.expectedErr,
+				)
```

Comment:
How about doing "string formatting" in the following way:

```suggestion
				t.Errorf("ConvertMetrics(%+v) = err(%v), want err(%v)",
					tc.input, err, tc.expectedErr)
```

I think it is also readable and consumes less space.

Reply:
The point of the formatter is automated consistency. I am open to making this change if you know a way to programmatically enforce it.

Reply:
I was not aware that it was done automatically